ordering items took 0:00:01
Tests considered 72 and ignored 317
2024-01-06 00:55:02,958 T0000: NOTICE WORKER = gw5
2024-01-06 00:55:02,958 T0000: NOTICE using testbed file /data/work/logs/docker-master-448374-l3/testbed_1_1_2.yaml
2024-01-06 00:55:02,958 T0000: NOTICE Execution Start Time: 2024-01-06 00:55:02.957235
2024-01-06 00:55:02,959 T0000: NOTICE LOGS PATH: /data/work/logs/docker-master-448374-l3/gw5
2024-01-06 00:55:03,579 T0000: NOTICE VERSION: UNKNOWN UNKNOWN UNKNOWN
2024-01-06 00:55:03,579 T0000: NOTICE HOST: sonicmgmtmaster MEM: 376gb 53.3%
2024-01-06 00:55:03,579 T0000: NOTICE ROOT_PATH: /repo
2024-01-06 00:55:03,580 T0000: NOTICE Python: 3.8.10
2024-01-06 00:55:03,580 T0000: NOTICE Topology unknown: D1T1:4
2024-01-06 00:55:03,580 T0000: DEBUG 
SUITE ARGS: --load-image none --max-time module 9000 --max-time function 5400 --max-time session 3600 --logs-level=debug --topology-check=function --env SPYTEST_TESTBED_IGNORE_CONSTRAINTS TG,TGCARD,TGSPEED,CHIP,MODEL --noop --env SPYTEST_RESET_CONSOLES 0 --noop --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 1 --noop --env SPYTEST_RECOVERY_CR_PER_CMD_POST_OP 1 --noop --env SPYTEST_BATCH_POLL_STATUS_TIME 300 --noop --env SPYTEST_BATCH_DEAD_NODE_MAX_TIME 7200 --noop --breakout-mode none --env SPYTEST_SHUTDOWN_FREE_PORTS 1 --env SPYTEST_HOOKS_BREAKOUT_UITYPE click --env SPYTEST_ONREBOOT_RENEW_MGMT_IP 0 --env SPYTEST_VERIFY_CSV_MIN_TOPOLOGY 1 --env SPYTEST_CMD_FAIL_RESULT_SUPPORT 0 --env SPYTEST_ROUTING_CONFIG_MODE split --env SPYTEST_CLEAR_DEVICE_METADATA_BGP_ASN 1 --env SPYTEST_TEXTFSM_USE_BUNDLED_PARSER 1 --feature-group master --ifname-type native --feature-disable bcmcmd --feature-disable warm-reboot --save config-db module --tclist-csv TestBGPConfed.test_confed_route_distribution_with_rmap,TestBGPConfed.test_ipv6_confed_route_distribution,TestBGPConfed.test_ipv6_confed_with_rr,TestBGPIPvxRouteAdvertisementFilter.test_bgp_ebgp4_nbr_authentication,TestBGPIPvxRouteAdvertisementFilter.test_bgp_ebgp4_nbr_update_source,TestBGPIPvxRouteAdvertisementFilter.test_bgp_route_map_with_community,TestBGPIPvxRouteAdvertisementFilter.test_default_originate_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_distribute_list_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_distribute_list_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_filter_list_out_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_prefix_list_out_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_prefix_list_out_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_route_aggregate_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_route_map_in_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_route_map_in_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_static_blackhole_rt_redistribute_with_routemap_ipv6,TestBGPL3Lag.test_ft_bgp_l3lag_peer_traffic_check,TestBGPRif.test_bgp_route_aggregation_4byteASN,TestBGPRif.test_ft_bgp_clear,TestBGPRif.test_ft_bgp_graceful_restart_and_aware_routers,TestBGPRif.test_ft_bgp_peer_traffic_check,TestBGPRif.test_ft_bgp_rmap,TestBGPRif.test_ft_bgp_v4_dyn_nbr,TestBGPRif.test_ft_bgp_v6_dyn_nbr,TestBGPRif.test_ft_bgp_v6_link_local_bgp,TestBGPRrTraffic.test_ft_bgp6_rr_traffic_check,TestBGPVeLag.test_ft_bgp_clear,TestBGPVeLag.test_ft_bgp_peer_traffic_check,test_VrfFun001_06,test_VrfFun_05_50,test_VrfFun_10_12_14,test_VrfFun_20_24_25_32_33_44_45,test_VrfFun_26_27,test_cli_validation_bgp_router_config,test_cli_validation_ip_address,test_ft_arp_dynamic_renew_traffic_test,test_ft_bgp_ebgp_community_map,test_ft_bgp_ibgp_RR_Loop,test_ft_ip6_static_route_traffic_forward_blackhole,test_ft_ip_static_ip_on_mgmt_intrf,test_ft_ip_static_route_traffic_forward,test_ft_ip_v4_v6_L2_L3_translation,test_ft_ipv6_neighbor_entry,test_ft_l3_performance_enhancements_v4_route_intstall_withdraw,test_ft_nat_docker_restart,test_ft_ping_v4_v6_after_ip_change_pc,test_ft_ping_v4_v6_vlan,test_ft_qos_config_mgmt_verifying_config_with_save_reboot,test_ft_sflow_max_sflow_collector_config,test_ft_sflow_sampling_v6_sFlow_collector,test_ft_system_config_mgmt_verifying_config_with_save_reboot_sflow,test_ft_verify_interfaces_order,test_l2_to_l3_port,test_l3_v4_route_po_1,test_vrf_bgp,test_vrf_reload,test_vrf_route_leak,test_vrf_scale,TestBGPRif.test_ft_bgp_ipv4_no_route_aggregation_for_exact_prefix_match,TestBGPRif.test_ft_bgp_ipv4_route_aggregation_atomic_aggregate_without_as_set,TestBGPRif.test_ft_bgp_ipv6_route_aggregation_with_as_set,TestBGPRif.test_ft_bgp_v4_max_dyn_nbr,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_connected_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_static_ipv4,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_connected_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_redistribute_static_ipv6,TestBGPIPvxRouteAdvertisementFilter.test_default_originate_ipv6,test_spytest_infra_first,test_spytest_infra_second,test_spytest_infra_last --noop

2024-01-06 00:55:03,580 T0000: NOTICE ARGS: --logs-level debug --results-prefix=results --get-tech-support none --fetch-core-files none --syslog-check none --save config-db module --topology-check module --logs-path logs/2024-01-06-00-50-16 --continue-on-collection-errors --max-time session 0 --max-time module 0 --max-time function 0 --noop --tryssh=0 --testbed-file testbed.yaml --env SPYTEST_RPS_DEBUG 0 --tclist-bucket 1,2,4 --test-suite community-vs-l3 --email rama.kristipati@broadcom.com --env SPYTEST_SCAPY_DBG_LVL 2 --noop --env SPYTEST_EMAIL_FROM SPyTest-VSNet@broadcom.com --noop --logs-path logs/docker-master-448374-l3 --env SPYTEST_EMAIL_BODY_PREFIX http://10.193.81.56:8080/sonic_mgmt_master/work/logs/docker-master-448374-l3/dashboard.html --email-subject-nargs VSNet-sonic_mgmt_master-DOCKER-L3-{{bldver}} : {{passrate}}: {{pass_cnt}}/{{executed}} docker-master-448374-l3 --noop --feature-group master
2024-01-06 00:55:04,243 T0000: DEBUG 'errors' not set in properties for V2 assuming 'default'
2024-01-06 00:55:04,244 T0000: INFO  TGEN-IP 127.0.0.1
2024-01-06 00:55:04,244 T0000: WARN  Registering Topology
2024-01-06 00:55:04,260 T0000: INFO  Connecting all devices
2024-01-06 00:55:04,264 T0000: INFO  Loading scapy:1.0 127.0.0.1 Ports: ['1/5', '1/6', '1/7', '1/8'] skip: False
2024-01-06 00:55:04,478 T0002: INFO  [D1-V2] Connecting to device (V2): sonic_ssh: 172.18.0.3:22 ..
2024-01-06 00:55:04,479 T0002: INFO  [D1-V2] TRY Authentication: 172.18.0.3:22 admin YourPaSsWoRd
2024-01-06 00:55:05,297 T0000: INFO  TG Scapy Init
2024-01-06 00:55:05,297 T0000: INFO  TG Base Init skip: False
2024-01-06 00:55:05,302 T0000: INFO  TG Scapy Connect 127.0.0.1:8009
2024-01-06 00:55:05,441 T0000: INFO  Server Logs Path /data/work/scapy-tgen/logs/current/inst-2024_01_06_00_55_05_330385/gw5/default.log
2024-01-06 00:55:05,447 T0000: DEBUG REQ: tg_connect(port_list=['1/5', '1/6', '1/7', '1/8'])
2024-01-06 00:55:09,711 T0002: DEBUG [D1-V2] Connected ...
2024-01-06 00:55:10,536 T0002: INFO  [D1-V2] Prompt at the connection start: 'admin@V2:~$' 
2024-01-06 00:55:10,538 T0002: AUDIT [D1-V2] ['export TMOUT=0', 'stty cols 5000', 'uptime']
2024-01-06 00:55:10,946 T0002: INFO  [D1-V2] FCMD: export TMOUT=0;stty cols 5000;uptime
2024-01-06 00:55:11,203 T0002: INFO  [D1-V2]  00:55:11 up 29 days,  2:00,  1 user,  load average: 9.85, 12.87, 11.23
2024-01-06 00:55:11,203 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:11,207 T0002: AUDIT [D1-V2] cat /proc/uptime
2024-01-06 00:55:11,613 T0002: INFO  [D1-V2] FCMD: cat /proc/uptime
2024-01-06 00:55:11,873 T0002: INFO  [D1-V2] 2512833.73 198068890.41
2024-01-06 00:55:11,873 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:11,878 T0002: INFO  [D1-V2] UPTIME: 2512833.73
2024-01-06 00:55:11,881 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:12,289 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:12,646 T0000: DEBUG RESP: tg_connect {"port_handle": {"1/5": "port-5", "1/6": "port-6", "1/7": "port-7", "1/8": "port-8"}, "status": 1}
2024-01-06 00:55:12,663 T0000: INFO  TG Connection: Success
2024-01-06 00:55:12,663 T0000: INFO  TG Connect...done
2024-01-06 00:55:13,052 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:13,053 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:13,055 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:13,057 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:13,057 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:18,060 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:18,466 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:19,126 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:19,126 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:19,127 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:19,128 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:19,128 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:24,132 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:24,540 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:25,252 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:25,252 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:25,254 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:25,255 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:25,255 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:30,260 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:30,667 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:31,377 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:31,377 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:31,380 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:31,382 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:31,382 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:36,384 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:36,792 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:37,607 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:37,608 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:37,612 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:37,613 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:37,613 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:42,616 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:43,022 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:43,733 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:43,733 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:43,736 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:43,737 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:43,737 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:48,740 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:49,148 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:49,810 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:49,810 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:49,813 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:49,815 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:49,815 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:55:54,820 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:55,227 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:55,889 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:55:55,889 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:55:55,892 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:55:55,894 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:55:55,894 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:00,900 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:01,308 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:02,020 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:02,021 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:02,024 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:02,026 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:02,026 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:07,028 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:07,436 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:08,097 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:08,097 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:08,100 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:08,101 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:08,101 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:13,104 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:13,512 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:14,172 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:14,172 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:14,174 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:14,175 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:14,175 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:19,181 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:19,588 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:20,250 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:20,250 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:20,253 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:20,255 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:20,255 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:25,260 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:25,665 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:26,428 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:26,428 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:26,430 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:26,432 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:26,432 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:31,438 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:31,851 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:32,563 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:32,564 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:32,569 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:32,572 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:32,572 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:37,576 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:37,984 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:38,644 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:38,644 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:38,647 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:38,648 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:38,648 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:43,652 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:44,059 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:44,719 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:44,720 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:44,722 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:44,724 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:44,725 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:49,728 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:50,137 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:50,798 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:50,799 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:50,801 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:50,803 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:50,803 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:56:55,808 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:56,213 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:56,921 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:56:56,921 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:56:56,922 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:56:56,924 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:56:56,924 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:01,929 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:02,333 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:03,041 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:57:03,041 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:57:03,043 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:03,044 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:57:03,044 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:08,049 T0002: AUDIT [D1-V2] sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:08,455 T0002: INFO  [D1-V2] FCMD: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:09,267 T0002: INFO  [D1-V2] Error response from daemon: No such container: bgp
2024-01-06 00:57:09,268 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:57:09,270 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_no_response_from_container command: sudo vtysh -c 'show running-config | include hostname'
2024-01-06 00:57:09,272 T0002: WARN  [D1-V2] Failed to read hostname from vtysh retry again in 5 sec
2024-01-06 00:57:09,272 T0002: INFO  Sleep for 5 sec(s)... @371
2024-01-06 00:57:14,277 T0002: ERROR [D1-V2] Failed to read hostname from vtysh - assuming sonic
2024-01-06 00:57:14,279 T0002: INFO  [D1-V2] Base Prompt: sonic
2024-01-06 00:57:14,280 T0002: DEBUG [D1-V2] Wait for system ready Ref: 866
2024-01-06 00:57:14,685 T0002: INFO  [D1-V2] FCMD: sudo md5sum /etc/spytest/remote/spytest-helper.py
2024-01-06 00:57:14,942 T0002: INFO  [D1-V2] 4522abaafb8e29890a8377a810228525  /etc/spytest/remote/spytest-helper.py
2024-01-06 00:57:14,942 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:57:14,945 T0002: WARN  [D1-V2] Execute 'sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300' with timeout 600 sec -- TRY 1
2024-01-06 00:57:14,946 T0002: INFO  [D1-V2] SCMD: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:18,778 T0002: WARN  [D1-V2] Skipped error checking in session init but detected pattern: containing_python_traceback command: sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:18,780 T0002: INFO  [D1-V2] 
2024-01-06 00:57:18,780 T0002: INFO  [D1-V2] Remote CMD: 'show platform summary'
2024-01-06 00:57:18,780 T0002: INFO  [D1-V2] Error: Failed to execute 'show platform summary' (out: '') (err: 'Traceback (most recent call last):
2024-01-06 00:57:18,780 T0002: INFO  [D1-V2]   File "/usr/local/bin/show", line 5, in <module>
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]     from show.main import cli
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]     if is_gearbox_configured():
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-06 00:57:18,781 T0002: INFO  [D1-V2]     app_db.connect(app_db.APPL_DB)
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2] 
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2] Remote CMD: 'cat /usr/share/sonic/device/None/None/port_config.ini | tail -1'
2024-01-06 00:57:18,782 T0002: INFO  [D1-V2] Remote CMD: 'sudo show interfaces status '
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2] Error: Failed to execute 'sudo show interfaces status ' (out: '') (err: 'Traceback (most recent call last):
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]   File "/usr/local/bin/show", line 5, in <module>
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]     from show.main import cli
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 307, in <module>
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]     if is_gearbox_configured():
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]        ^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:18,783 T0002: INFO  [D1-V2]   File "/usr/local/lib/python3.11/dist-packages/show/main.py", line 248, in is_gearbox_configured
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2]     app_db.connect(app_db.APPL_DB)
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2]   File "/usr/lib/python3/dist-packages/swsscommon/swsscommon.py", line 1821, in connect
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2]     return _swsscommon.SonicV2Connector_Native_connect(self, db_name, retry_on)
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2] RuntimeError: Unable to connect to redis: Cannot assign requested address')
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2] 
2024-01-06 00:57:18,784 T0002: INFO  [D1-V2] admin@V2:~$ 
2024-01-06 00:57:18,785 T0002: ERROR [D1-V2] Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  
2024-01-06 00:57:18,785 T0002: ERROR exception in thread: Traceback (most recent call last):
  File "/repo/utilities/parallel.py", line 159, in _thread_func
    retvals[index] = func(*args, **kwargs)
  File "/repo/spytest/net.py", line 1064, in connect_to_device
    show_ver_output = self.connect_to_device_real(access, retry, recon, recover)
  File "/repo/spytest/net.py", line 1290, in connect_to_device_real
    self.do_common_init(devname, phase=0, max_ready_wait=max_ready_wait)
  File "/repo/spytest/net.py", line 866, in do_common_init
    sys_status = self.wa.wait_system_status(devname, max_time=max_ready_wait)
  File "/repo/spytest/framework.py", line 1616, in wait_system_status
    return self._wait_for_ports(dut, max_time)
  File "/repo/spytest/framework.py", line 1605, in _wait_for_ports
    return self.net.apply_remote(dut, "wait-for-ports", [max_time])
  File "/repo/spytest/net.py", line 4523, in apply_remote
    raise ValueError(msg)
ValueError: Failed to execute the command sudo python /etc/spytest/remote/spytest-helper.py --wait-for-ports 300  

2024-01-06 00:57:18,785 T0002: ERROR [D1-V2] Failed to connect to device
2024-01-06 00:57:18,786 T0000: ERROR Failed to connect (['V2'])
2024-01-06 00:57:18,786 T0000: INFO  --FINISH--RUN--4 @559
